### PR TITLE
Rephrase definition of single-expression function

### PR DESCRIPTION
**Pull request by vegarsti**
The current definition says "When a function returns a single expression", which I don't think is correct: As long as the return type is not `Void`, a function always returns a single expression. And it sounds as if the function returns all the expressions inside it.

This is an opinionated nitpick, though, so feel free to close this!

### CODE REVIEW

File: docs/topics/functions.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ fun printHello(name: String?) { ... }
 
 ### Single-expression functions
 
-When a function returns a single expression, the curly braces can be omitted and the body is specified after a `=` symbol:
+When a function contains a single `return` expression, the curly braces can be omitted and the body is specified after a `=` symbol:
```
Comment (sarahhaggarty) [inline docs/topics/functions.md:180]:
```suggestion
When the function body consists of a single expression, the curly braces can be omitted and the body specified after an `=` symbol:
```

**Review by sarahhaggarty — CHANGES_REQUESTED** [review]
I agree that using "returns" is confusing. What do you think about this version that I suggest instead?

**Review by sarahhaggarty — APPROVED** [review]

**Comment by sarahhaggarty** [comment]
Thanks for sharing your thoughts with us @vegarsti !

File: docs/topics/functions.md (hunk 1 above)
Comment (Cesar556) [reply docs/topics/functions.md]: - []()@_********_

**Pull request merged**